docs/commerce/checkout/configure_checkout.md

```diff
@@ -0,0 +1,195 @@
+
+``` yaml
+framework:
+    workflows:
+        ibexa_checkout:
+            type: state_machine
+            audit_trail:
+                enabled: false
+            marking_store:
+                type: method
+                property: status
+            supports:
+                - Ibexa\Contracts\Checkout\Value\CheckoutInterface
+            initial_marking: initialized
+            places:
+                - initialized
+                - address_selected
+                - shipping_selected
+                - summarized
+                - completed
+            transitions:
+                select_address:
+                    from:
+                        - initialized
+                        - address_selected
+                        - shipping_selected
+                        - summarized
+                    to: address_selected
+                    metadata:
+                        next_step: select_shipping
+                        controller: Ibexa\Bundle\Checkout\Controller\CheckoutStep\AddressStepController::renderStepView
+                        label: 'Billing & shipping address'
+                        translation_domain: checkout
+                select_shipping:
+                    from:
+                        - address_selected
+                        - shipping_selected
+                        - summarized
+                    to: shipping_selected
+                    metadata:
+                        next_step: summary
+                        controller: Ibexa\Bundle\Checkout\Controller\CheckoutStep\ShippingStepController::renderStepView
+                        label: 'Shipping & payment method'
+                        translation_domain: checkout
+                summary:
+                    from:
+                        - shipping_selected
+                        - summarized
+                    to: summarized
+                    metadata:
+                        next_step: complete
+                        controller: Ibexa\Bundle\Checkout\Controller\CheckoutStep\SummaryStepController::renderStepView
+                        label: 'Payment & summary'
+                        translation_domain: checkout
+                complete:
+                    from: summarized
+                    to: completed
+                    metadata:
+                        controller: Ibexa\Bundle\Checkout\Controller\CheckoutStep\CompleteStepController::renderCompleteView
+                        label: 'Order confirmation'
+                        translation_domain: checkout
+```
+
+## Configure shipping and payment methods
+
+You can define the shipping and payment methods.
+Under `ibexa.repositories.<repository_name>.checkout`, create entries that resemble 
+the following example:
+
+``` yaml 
+ibexa:
+    repositories:
+        <repository_name>:
+            checkout:
+                workflow: <your_workflow_identifier> #optional, "ibexa_checkout" is used by default
```

Comment:
This specific key should probably be mentioned in the section above, it doesn't refer to shipping/payment methods.

Reply:
Definitely, fixed